### ISSUE TITLE
Menubar - keyboard navigation enter doesn't work

### ISSUE THREAD

**Issue by geromegrignon** [openng-org/optimus-ui#583]
### Describe the bug

There is an issue with keyboard navigation in menubar when using custom template for menu item. We are unable to trigger menu command or route navigation using enter or space key. Event is blocked somewhere even if we add (keydown.enter) handler inside custom template.

### Reason for not contributing a PR

- [ ] Lack of time
- [x] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://stackblitz.com/edit/hcdxezjf?file=src%2Fapp%2Fimports.ts,src%2Fapp%2Fmenubar-template-demo.ts,src%2Fapp%2Fmenubar-template-demo.html

### Environment

Win 11

### Angular version

19.0.1

### PrimeNG version

v19

### Steps to reproduce the behavior

1. Go to https://stackblitz.com/edit/hcdxezjf?file=src%2Fapp%2Fimports.ts,src%2Fapp%2Fmenubar-template-demo.ts,src%2Fapp%2Fmenubar-template-demo.html
2. Focus on menubar.
3. Naviagate throught menu using left arrow, right arrow, down arrow.
4. Try to trigger command on any menu item using enter. 

You should see console log with label in console but nothing appears. But if you click on menu item with mouse, console log is triggered.

### Expected behavior

User should be able to trigger enter event or command of MenuItem using only keyboard.

> Migrated from `primefaces/primeng#17672` — originally by `@GitJacek` on 2025-02-13

**Comment by geromegrignon** [comment]
*[Originally posted by `@GitJacek` on 2025-02-26T14:19:15.000Z]*

Any chance it get some attention? This seems crucial for keyboard only users.

**Comment by geromegrignon** [comment]
*[Originally posted by `@kimondev` on 2025-05-05T06:51:16.000Z]*

Is there an update according to this issue? Or a workaround

**Comment by geromegrignon** [comment]
*[Originally posted by `@GitJacek` on 2025-05-05T07:21:24.000Z]*

I haven't found any workaround to add keyboard accessibility to the current component...

**Comment by geromegrignon** [comment]
*[Originally posted by `@jonasbeltoft` on 2025-08-13T21:39:01.000Z]*

Would love an update on this

**Comment by geromegrignon** [comment]
*[Originally posted by `@github-actions[bot]` on 2026-02-10T00:33:08.000Z]*

This issue has been automatically marked as stale due to a lack of activity. **If this issue is still relevant to you, please leave a comment** so we can keep it open. We apologize for not being able to prioritize it sooner. If you have any new information or questions, please share them in your comment!

**Comment by gianluca-moro** [comment]
Still an issue in v22

**Comment by geromegrignon** [comment]
Hi @GitJacek,

We regenerated your old issue as part of our fork of the original repo.
The original PrimeNG is now archived as explained here: https://primeui.dev/nextchapter

Your issue has been triaged by the community and identified as still relevant.

We'll have it addressed and you are also very welcomed to contribute by submitting a pull request.

Thanks for bringing the problem to our attention.
